src/Kentico.Xperience.Algolia/Indexing/AlgoliaOptions.cs

```diff
@@ -1,6 +1,6 @@
-﻿using Kentico.Xperience.Algolia.Services;
+﻿using Kentico.Xperience.Algolia.Search;
 
-namespace Kentico.Xperience.Algolia.Models
+namespace Kentico.Xperience.Algolia.Indexing
```

Comment:
I need to check with the architects if `xperience.algolia` is the appsetting section we should be using.

Reply:
They got back to me and said we should have a parent section using the `CMS` prefix and nest settings under that.

```json
{
    "ConnectionStrings": {
        // ...
    },

	"CMSAlgoliaSearch": {
	    "ApplicationId": "...",
        "CrawlerApiKey": "...",
        // ...
	}
}
```

Reply:
Changed

Reply:
Thanks 👍.

Can you use convert this to a file-scoped namespace?

Reply:
I have changed AlgoliaOptions.cs to file-scoped namespace

Reply:
Changed in AlgoliaOptions.cs